docs/beta/plugins/discord.md

```diff
@@ -1,5 +1,248 @@
+
+The Discord plugin creates the following types of nodes in the contribution graph:
+
+- **User**:
+
+A Discord user account, e.g. [@decentralion {SB: correct link?}](). User accounts do not mint Cred, so setting a node weight would have no effect. Using the identity plugin, it's possible to "collapse" user nodes with other identity nodes into a single, canonical identity. For example, if a contributor had a Discord user account and a GitHub account, then the identity plugin can collapse those identities together.
+
+Users are connected to messages they author, to messages they react to, and to messages that mention them {SB: is this true?}.
+
+- **Guild Member**:
+
+A Discord user account, e.g. [@decentralion {SB: correct?}](), that has special powers. E.g. their reactions mint more Cred {SB: totally making this up here. Is this part of beta launch, or just a hidden alpha feature for MetaGame?}
+
+- **Bot**:
+
+{SB: below is copied from GitHub page. Still true for Discord? Do we support Discord bots?}
+A Discord user account that has been explicitly marked as a bot, via inclusion in bots.js. This is useful so we can filter out bot accounts from receiving grain or showing up in the Cred rankings.
+
+Bots have the same connections as users.
+
+- **Message**:
+
+A Discord message, e.g. ["Welcome to the SourceCred community!..."](https://discordapp.com/channels/453243919774253079/715770410955964477/715776215528505385) {SB: this is a link to the welcome message in the SourceCred Discord #start-here channel}. A message node will be connected to its author (user node creating the message), emoji reactions to the message, and any references to the message {SB: is this true??}. 
+
+[INSERT SCREENSHOT MESSAGE]
+
+- **Channel**:
+
+A Discord channel, e.g. [#start-here](https://discord.gg/qadM9u) {SB: link needs checking. Got link by clicking 'Create Invite' on the channel, but getting 'invalid link' msg for some reason when I try it locally). A channel node will be connected to all messages in that channel. 
+
+Found this in the code and pasting here in case the plugin exposes these channel types. 
+
+```
+ * Discord Channels can be of various types, defined below.
+ * See: https://discordapp.com/developers/docs/resources/channel#channel-object-channel-types
+ */
+export type ChannelType =
+  | "GUILD_TEXT"
+  | "DM"
+  | "GUILD_VOICE"
+  | "GROUP_DM"
+  | "GUILD_CATEGORY"
+  | "GUILD_NEWS"
+  | "GUILD_STORE";
+```
+
+- **Reaction**:
+
+A Discord emoji reaction, e.g. [insert cute emoji reaction code]. When a user reacts to a message, a reaction node is created that is connected to the author of the reaction (user that reacted) and the message reacted to. 
+
+
+[INSERT SCREENSHOT REACTIONS TO MESSAGE]
+
+
+### Edges
+
+The Discord plugin creates the following kinds of edges (connections between nodes) 
+in the contribution graph:
+
+- **Authors**:
+
+An "authors" edge connects an author (user or bot) to a message they created. 
+
+- **References**: 
```

Comment:
There's no references in the discord plugin currently, the closest thing would be "mentions" which is when you @ someone, it creates an edge from that message to the person you mentioned.

Reply:
So is this a new edge type? Or is it a 'Reacts to' or 'Adds reaction', and I just need to update the definitions of those?